### PR TITLE
IconButton: now with less IE11 pixel shift

### LINKED ISSUE #5847: Icon Button: Clicking on the button creates flickering behaviour
### Bug Report 
 
- __Package version(s)__: (fill this out)
- __Browser and OS versions__: IE and Windows 10
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? No
 
Requested priority: Normal
 
Products/sites affected: VSTS
 
#### Describe the issue:
 
Clicking on the button makes the button jumps. 
 
#### Actual behavior:
 
The button jumps when clicked. 
 
#### Expected behavior:
 
The button should not show flickering/jumpy behavior on click. 
 
iconbuttonflickerbehaviour

### PR DESCRIPTION
**Pull request by dzearing**
#### Pull request checklist

- [X] Addresses an existing issue: Fixes #5847
- [X] Include a change request file using `$ npm run change`

#### Description of changes

Addressing button content styling to fix IE11 pixel shifting.

Also fixing the `npm start` demo site for IE11 as the Promise polyfill was missing.

(optional)

### CODE REVIEW

File: common/changes/@uifabric/fabric-website-resources/button-fix_2018-08-23-03-25.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/fabric-website-resources",
+      "comment": "IE11 Promise polyfill was missing and broke `npm start`. Added polyfill.",
```
Comment (cliffkoh) [inline common/changes/@uifabric/fabric-website-resources/button-fix_2018-08-23-03-25.json]: :+1:

**Comment by micahgodbolt** [comment]
kicked off new build

**Comment by micahgodbolt** [comment]
needs new snapshots

**Comment by dzearing** [comment]
oops will get this in.

**Pull request merged**